macOS/Configuration/App/DuckDuckGoAppStore.xcconfig

```diff
@@ -33,7 +33,14 @@ ENABLE_APP_SANDBOX = YES
 MACOSX_DEPLOYMENT_TARGET = 12.3
 
 PRODUCT_MODULE_NAME = $(APP_STORE_PRODUCT_MODULE_NAME_OVERRIDE:default=$(DEFAULT_PRODUCT_MODULE_NAME))
-PRODUCT_NAME_PREFIX = DuckDuckGo App Store
+PRODUCT_NAME_PREFIX[config=Debug][arch=*][sdk=*] = DuckDuckGo App Store
+PRODUCT_NAME_PREFIX[config=CI][arch=*][sdk=*] = DuckDuckGo App Store
+PRODUCT_NAME_PREFIX[config=Review][arch=*][sdk=*] = DuckDuckGo App Store
+PRODUCT_NAME_PREFIX[config=Release][arch=*][sdk=*] = DuckDuckGo App Store
+
+PRODUCT_NAME[config=Debug][arch=*][sdk=*] = $(PRODUCT_NAME_PREFIX)
+PRODUCT_NAME[config=CI][arch=*][sdk=*] = $(PRODUCT_NAME_PREFIX)
+PRODUCT_NAME[config=Review][arch=*][sdk=*] = $(PRODUCT_NAME_PREFIX)
```

Comment:
Nothing really changed here I believe - I only expanded these to cover all configs and it seems that did the trick at least for DEBUG usage.